tests/test_file_filter.py

```diff
@@ -87,6 +87,42 @@ def test_get_file_filter_terms():
     assert filter_terms == ["file 1\\.org", "/path/to/dir/.*\\.org"]
 
 
+def test_exclude_file_filter_terms():
+    # Arrange
+    file_filter = FileFilter()
+    q_with_filter_terms = 'head tail file:-"file 1.org" file:"/path/to/dir/*.org"'
+
+    # Act
+    filter_terms = file_filter.get_filter_terms(q_with_filter_terms)
+
+    # Assert
+    assert filter_terms == ["file 1\\.org", "/path/to/dir/.*\\.org"]
+
+
+def test_file_exclude_filter():
+    # Arrange
+    file_filter = FileFilter()
+    q_with_exclude_filter = 'head -"exclude_file" tail'
+
+    # Act
+    can_filter = file_filter.can_filter(q_with_exclude_filter)
+
+    # Assert
+    assert can_filter == True
+
+
+def test_file_include_and_exclude_filter():
+    # Arrange
+    file_filter = FileFilter()
+    q_with_include_and_exclude_filter = 'head "include_file" -"exclude_file" tail'
```

Comment:
The query tested seems to be for word filters instead of file filters. Wouldn't the query with file filter be of the form:
```suggestion
    q_with_include_and_exclude_filter = 'head file:"include_file" -file:"exclude_file" tail'
```